source/experimental.html.md.erb

```diff
@@ -1932,9 +1932,9 @@ The type of error that is being reported.
 |provider_id|[UUID](#schemauuid)|false|none|[Universally unique identifier.](https://en.wikipedia.org/wiki/Universally_unique_identifier)|
 |reference|string|false|none|none|
 |custom_data|[CustomData](#schemacustomdata)|false|none|Use this to store any state required by later requests relating to this check. The intent is for your integration to be completely stateless itself, to make it simpler to deploy and maintain. Note: if present, this *must* be an object, rather than an array or other JSON value. However, the keys and values in this object can be anything you like, although we don't recommend storing large amounts of data here, as it will be passed in on every following request to your integration relating to this check.|
-|provider_data|[ProviderData](#schemaproviderdata)|false|none|The structured JSON data returned by the data provider, or a representation of the provider's response, to allow PassFort to investigate issues that may arise with your integration.|
-|warnings|[Warnings](#schemawarnings)|false|none|Similar to 'errors' except that the sub_type and data fields are not present and providing warnings will not cause PassFort to consider the check as errored.|
-|errors|[Errors](#schemaerrors)|false|none|A list of errors that occurred while running this check. If this is provided and it's not empty, the result of this check will be considered to be errored by PassFort, regardless of the other contents of this response.|
+|provider_data|[ProviderData](#schemaproviderdata)|false|none|The structured JSON data returned by the data provider, or a representation of the provider's response, to allow Passfort to investigate issues that may arise with your integration.|
+|warnings|[Warnings](#schemawarnings)|false|none|Similar to 'errors' except that the sub_type and data fields are not present and providing warnings will not cause Passfort to consider the check as errored.|
```

Comment:
As above, link "errors" to relevant section, format <code>sub_type</code> as code, and change "are not" to "aren't" and "will not" to "won't"